src/main/java/org/schemaspy/model/Html.java

```diff
@@ -0,0 +1,108 @@
+package org.schemaspy.model;
+
+import org.schemaspy.Config;
+import org.schemaspy.cli.CommandLineArguments;
+
+import java.io.File;
+import java.util.Collection;
+
+/**
+ * Decorator for rendering console-based details for HTML.
+ */
+public class Html implements ProgressListener {
```

Comment:
Most methods are used by Html generating Code. But what it does is write to System.out which in it self isn't great.

System.out is StdOut so may be StdOut or Console.